dashing/ci-nightly-debug.yaml

```diff
@@ -12,8 +12,10 @@ build_environment_variables:
 build_tool: colcon
 build_tool_args: '--cmake-args -DCMAKE_BUILD_TYPE=Debug --no-warn-unused-cli'
 install_packages:
+- default-jdk # for CycloneDDS
```

Comment:
The debug and release builds are supposed to only test the tier 1 RMW implementation. So CycloneDDS shouldn't be added to them. So instead of adding these dependencies CycloneDDS should be added to the `--packages-ignore`.

Reply:
Fixed in 4c046a1, and opened #47.

Reply:
Would it be appropriate to target the extra RMW build for the overlay job then?

Reply:
For CycloneDDS that would work since it doesn't need a custom typesupport. For OpenSplice it wouldn't since the message packages in the underlay haven't generated the OpenSplice specific typesupport.

Reply:
Why wouldn't it, if the underlay was the extra RMW build? That builds OpenSplice.

Reply:
Oh, that would work fine. I was mislead by the context here and thought it builds on top of the debug one.